### PR TITLE
little mdbook updates

### PR DESCRIPTION
**Pull request by Connoropolous**

### CODE REVIEW

**Comment by zippy** [comment]
Hey @Connoropolous lots of concurrent building going on so please remember on docs PRs to cancel builds!

File: doc/holochain_101/src/planning_a_dapp.md [hunk 1]
```diff
@@ -34,16 +34,16 @@ One must also re-think the dApp's business model such that it does not rely on a
...
+Hashes ensure the reliability of information by representing any given piece of data with a unique, consistent string of random looking characters. This makes changes to data visible because one can see that a hash has changed without needing to inspect the data itself.  
 
-However, it is impossible to get the original data from a hash -- its purpose is to prove that the data to which it corresponds has not been altered. The same data consistently gives the same hash, and different data always gives a completely different hash. 
+However, it is impossible to get the original data from a hash -- its purpose is to prove that the data to which it corresponds has not been altered. The same data consistently gives the same hash, and different data always give a completely different hash. 
 
 These features imply that one can use small, portable hashes to verify data. One could also use a database containing data and their hashes as a table of contents, indexing (though not reading) data associated with a given hash.
```
Comment (zippy) [inline doc/holochain_101/src/planning_a_dapp.md]:
```suggestion

In the context of Holochain hashes are frequently used to look up content, both in our internal implementations as well as on the DHT.  Therefore we frequently refer to the hash of some item (i.e. an entry on the chain) as its *Address*.
```

File: doc/holochain_101/src/planning_a_dapp.md [hunk 2]
```diff
@@ -52,21 +52,25 @@ Encryption is a two way process, so the right key enables one to decrypt an encr
...
 ### Public, shared data on a public network
 
 Public data works like Bittorrent: 
 
-Anybody can join a network
-anybody can request any data they want from the network
-any data is available as long as at least one person is sharing it
-if some data is not shared by enough people, a new random person on the network must share it
-there is no "local only" data  
+   anybody can join a network
```
Comment (zippy) [inline doc/holochain_101/src/planning_a_dapp.md]: use bullets here, i.e.:

**Comment by thedavidmeister** [comment]
@Connoropolous is this still WIP or ready for review?

**Comment by Connoropolous** [comment]
@thedavidmeister I'm continuing to do a bunch of writing for the book, and will be for the next week or so, and then I'll update this PR I think :)

**Comment by zippy** [comment]
@Connoropolous I'd like to get as many PRs merged or cleaned up as possible, to reduce PR noise.  Can we either merge this one, or can you merge it into other mdbook prs?

**Pull request merged**